src/helpers/api.js

```diff
@@ -1,6 +1,12 @@
 import axios from 'axios';
 import store from '@/store/index';
 import apiUrls from '@/setup/api-urls.json';
+import { i18n } from '@/setup/i18n';
+
+// DRY: Define axios instance with custom config applied to all requests.
```

Comment:
suggestion: Please don't use abbreviations when ever possible. Not everyone is familiar with words like `DRY`. I would recommend a comment like `Creating an Axios instance to set general header properties (for each request).`